docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/fluentd/page.mdx

---
title: 'Fluentd'
description: 'Forward logs and events from Fluentd into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# Fluentd

Fluentd is an open-source data collector that aggregates logs and events from multiple sources into a unified pipeline. It provides flexible routing, transformation, and integration with hundreds of data backends.

## Using Fluentd with GlassFlow

GlassFlow can ingest telemetry forwarded by Fluentd into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Fluentd connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Fluentd connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
